aviary/mission/flight_phase_builder.py

```diff
@@ -170,6 +175,37 @@ def build_phase(self, aviary_options: AviaryValues = None, phase_type=EquationsO
             rate_targets = ['dh_dr']
             rate2_targets = ['d2h_dr2']
 
+        # For multi-engine cases, we may have throttle allocation control.
+        if phase_type is EquationsOfMotion.HEIGHT_ENERGY and num_eng > 1:
+            allocation = user_options.get_val('throttle_allocation')
+
+            # Allocation should default to an even split so that we don't start
+            # with an allocation that might not produce enough thrust.
```

Comment:
Are you talking about val? That's just the default value, although default throttle seems to get higher the more engines there are (2 engines defaults them to 0.5 ea, 3 engines defaults them to 0.6667 ea, etc.) @Kenneth-T-Moore what is the logic behind the pattern?

Odd numbers of engines shouldn't cause any problems, val is just a initial value and the shape is always number of engines types - 1 (last is picked w/ solver) so it is fully continuous

Reply:
This is a per-engine-type basis.

2 models:  0.5, 0.5
3 models: 0.33, 0.33, 0.33

I found that if I didn't include an initial starting point for the allocation, it would start at (0, 1.0), or (0, 0, 1.0). These are more degenerate because you might not have enough thrust with one or more engine types turned off, and the optimization would be more tenuous from there.  Specifically, the thrust balance would command a throttle on the single allocated engine to be greater than 1, so you would start with that constraint violated. Depending on the engine model, you could be so far in the junk extrapolation zone that you start out in inelastic mode in SNOPT.

Reply:
Ah I missed the np.ones() in this logic so I was confused on the allocation math (saw it as just (num_engs-1)/num_eng). Makes sense now!

Reply:
I can't remember what my specific concern was, but it might have been thinking that those were the values that would be used for the STATIC allocation, but if these are just an initial guess, that should be fine.